File: src/main/java/io/github/cdgeass/editor/dom/element/mapper/CacheRef.java
package io.github.cdgeass.editor.dom.element.mapper;

import com.intellij.util.xml.DomElement;
import com.intellij.util.xml.GenericAttributeValue;

/**
 * @author cdgeass
 * @since  2020-06-20
 */
public interface CacheRef extends DomElement {

    GenericAttributeValue<String> getNamespace();
}

File: src/main/kotlin/io/github/cdgeass/codeInsight/dom/element/CacheRef.kt
// Generated on Fri Mar 26 00:14:00 CST 2021
// DTD/Schema  :    mybatis-3-mapper.dtd

package io.github.cdgeass.codeInsight.dom.element

import com.intellij.util.xml.*

/**
 * mybatis-3-mapper.dtd:cache-ref interface.
 * @author cdgeass
 */
interface CacheRef : DomElement {

    /**
     * Returns the value of the namespace child.
     * Attribute namespace
     * @return the value of the namespace child.
     */
    @Required
    @Attribute("namespace")
    @Resolve(Cache::class)
    fun getNamespace(): GenericAttributeValue<Cache>
}
